stories/SQForm.stories.js

```diff
@@ -526,6 +526,7 @@ export const applyAnAction = () => {
           label="Actions with a default value"
           size={5}
           isRequired={true}
+          isDisabled
```

Comment:
@Chris-Boyle Could you make this boolean value a knob in storybook the user can toggle enable/disabled state, please?

Reply:
Yes!